chapters/descriptorsets.txt

```diff
@@ -2584,6 +2584,10 @@ ifdef::VK_VERSION_1_1,VK_KHR_maintenance1[]
     pname:imageView must: not be 2D or 2D array image view created from a 3D
     image
 endif::VK_VERSION_1_1,VK_KHR_maintenance1[]
+  * If pname:imageView is created from a depth/stencil image the
+    pname:aspectMask used to create the pname:imageView must: include either
+    ename:VK_IMAGE_ASPECT_DEPTH_BIT or ename:VK_IMAGE_ASPECT_STENCIL_BIT but
```

Comment:
(Readding as I am confident now it is only spec bug #714):

This could be simplified somehow as depth\stencil image views are already supposed to be `VK_IMAGE_ASPECT_DEPTH_BIT` and\or `VK_IMAGE_ASPECT_STENCIL_BIT`.